scipy/diff/numdifftools/tests/test__step_generators.py

```diff
@@ -0,0 +1,46 @@
+import numpy as np
+from numpy.testing import (assert_, run_module_suite)
+from scipy.diff.numdifftools._step_generators import _generate_step
+
+
+def steps(base_step, step_ratio, num_steps, sign):
+    for i in range(num_steps):
+            steps = base_step * step_ratio ** (sign*i)
+            if (np.abs(steps) > 0).all():
+                yield steps
+
+
+class Test(object):
+    def test_min_step(self):
```

Comment:
These tests are probably not super helpful - the ``steps`` generator above is pretty much an exact copy of the code it is supposed to test. Is there a way you can predict what the answer should be, and then check against that? (e.g. compare ``_generate_step(..., num_steps=5)`` with ```[1, 2, 4, 8, 16]``)